### PR TITLE
Added customizability  

### PR DESCRIPTION
**Pull request by stha-ums**
I hope these features would be beneficial.

Added Features:

1. Height parameter for calendar cell height and header height,
2. Builder function for empty cell builder, (empty means the cell that does not contain date )
3. exposed call back function on month change.  

simple possibility,

0-02-03-f6f3344a84e17758a74244a762458bd63c313fb7fcb36a714b7499f6fddfb91a_206685174312da

### CODE REVIEW

**Comment by lohanidamodar** [comment]
@stha-ums Wow, great going! Also, next time work on develop and Open PRs on develop branch.

**Comment by stha-ums** [comment]
Sure

**Review by lohanidamodar — APPROVED** [review]

**Review by lohanidamodar — CHANGES_REQUESTED** [review]
बिहिबार हुनु पर्ने बिहिवार भएछ same for शनिबार

**Comment by stha-ums** [comment]
वार hunxa, not बार

**Comment by lohanidamodar** [comment]
@stha-ums I think बार नै हुन्छ not वार , confirm with dictionary too..

File: lib/src/days_view.dart [hunk 1]
```diff
@@ -228,11 +231,11 @@ final List<String> dayHeaderFullNameEnglish = [
 final List<String> dayHeaderFullNameNepali = [
   "आइतबार",
   "सोमबार",
-  "मन्गलबार",
+  "मंगलवार",
```
Comment (lohanidamodar) [inline lib/src/days_view.dart]: should be मंगलबार

File: lib/src/days_view.dart [hunk 2]
```diff
@@ -228,11 +231,11 @@ final List<String> dayHeaderFullNameEnglish = [
...
   "बुधबार",
-  "बिहीबार",
+  "बिहिवार",
```
Comment (lohanidamodar) [inline lib/src/days_view.dart]: बिहिबार

File: lib/src/days_view.dart [hunk 3]
```diff
@@ -228,11 +231,11 @@ final List<String> dayHeaderFullNameEnglish = [
...
   "शुक्रबार",
-  "शनीबार"
+  "शनिवार"
```
Comment (lohanidamodar) [inline lib/src/days_view.dart]: शनिबार

**Pull request closed without merging**